drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -125,10 +125,40 @@ def student_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
-        fragment.initialize_js('DragAndDropBlock')
+        fragment.initialize_js('DragAndDropBlock', self.get_configuration())
 
         return fragment
 
+    def get_configuration(self):
+        """
+        Get the configuration data for the student_view.
+        The configuration is all the settings defined by the author, except for correct answers
+        and feedback.
+        """
+
+        def items_without_answers():
+            items = copy.deepcopy(self.data.get('items', ''))
+            for item in items:
+                del item['feedback']
+                del item['zone']
```

Comment:
@bradenmacdonald What do you think about making `zone` more specific? Based on the name alone it's not entirely clear (to me) that the value associated with this key represents the zone that the item should be placed on to obtain a correct answer. Using something like `target_zone` would help I think.